lib/mailgun.rb

```diff
@@ -20,7 +20,10 @@ module Mailgun
   def send_message(**params)
     make_valid!(params)
     text = params[:text]
+    stop_this_notification_url = params[:stop_this_notification_url]
+    unsubscribe_url = params[:unsubscribe_url]
     params[:text] = Social.email_template.get binding
```

Comment:
Use an if statement?

Reply:
can you explain what will happen if i do not use an if statement? i guess if they are nil, mailgun will send text without unsubscribe link because i put if in 
```
params[:text] = Social.email_template_with_unsubscribe_urls.get binding if params[:unsubscribe_from_all_notifications_url].present?
```

Reply:
```ruby
params[:text] = params[:unsubscribe_url].present? ? Social.email_template_with_unsubscribe_link.get binding : Social.email_template.get binding
```

Something like this?